### PR TITLE
 Align What is Your NINO Screen with MVP Designs

### PR DESCRIPTION
**Pull request by liamStockwellGDS**
OJ-2414 NINO | FE | Align What is Your NINO Screen with MVP Designs

### What changed

updated Locales with updated wording,
updated validator and controller, to handle new nino input behaviour
updated 2 views to reflect the changes in ui

### Why did it change

changes to the controller and Validator, were to handle whitespace and lowercase letters

### Issue tracking

[OJ-2414] (https://govukverify.atlassian.net/browse/OJ-2414)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[OJ-2414]: https://govukverify.atlassian.net/browse/OJ-2414?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/locales/cy/fields.yml [hunk 1]
```diff
@@ -48,4 +49,7 @@ abandonWarning:
   content: "Ni allwch ddychwelyd i'r sgrin hon os ydych yn chwilio am ffordd arall o brofi eich hunaniaeth."
 
 abandonLink:
-  content: "<a href='/check/abandon' class='govuk-link' id='abandon-link' data-id='abandon'> I do not have my NINO </a>"
+  content: "<a href='/check/abandon' class='govuk-link' id='abandon-link' data-id='abandon'> Nid yw fy rhif Yswiriant Gwladol gennyf </a>"
+
+continueButton:
+  content: "…Parhau"
```
Comment (Caitlin-cooling) [inline src/locales/cy/fields.yml:55]: Not sure this should have `...` in it?

File: src/views/check/could-not-match-national-insurance.njk [hunk 2]
```diff
@@ -7,13 +7,18 @@
...
+
+    <p class="govuk-body" >{{'could-not-match-national-insurance.paragraph2' | translate }}</p>
+
   {% call hmpoForm(ctx) %}
       {{ hmpoRadios(ctx, {
             id: "retryNationalInsuranceRadio",
             namePrefix: "retryNationalInsuranceRadio",
             fieldset: {
                 legend: {
-                    isPageHeading: true,
```
Comment (Caitlin-cooling) [inline src/views/check/could-not-match-national-insurance.njk:16]: Now that we've removed the override we can actually probably not rely on the defaults from the form wizard which means the `block mainContent` can be removed and the `paragraph1` `paragraph2` stuff can be changed back to `content`? Or am I missing something?

File: src/locales/cy/fields.yml (hunk 1 above)
Comment (liamStockwellGDS) [reply src/locales/cy/fields.yml]: the translations came back and this has now been changed to "Gwirio a pharhau"

File: src/locales/en/fields.yml [hunk 3]
```diff
@@ -1,10 +1,11 @@
 nationalInsuranceNumber:
   content: " "
-  label: What is your national insurance number?
-  hint: It's on your National Insurance card, benefit letter, payslip or P60. For example, 'QQ123456C'
+  label: "National Insurance number"
+  hint: "For example, ‘QQ 12 34 56 C’."
```
Comment (Caitlin-cooling) [inline src/locales/en/fields.yml:4]: I'm being really nitpicky here but there is a full stop in English and not Welsh, can we align them?

**Review by Caitlin-cooling — DISMISSED** [review]
Nice work! Like to see that we've been able to remove some code

**Pull request merged**